python/cudf/cudf/core/indexed_frame.py

```diff
@@ -2649,6 +2697,705 @@ def _explode(self, explode_column: Any, ignore_index: bool):
+        triangle        3      180
+        rectangle       4      360
+            """,
+            ser_op_example=""">>> a.multiply(b)
+        a       1
+        b    <NA>
+        c    <NA>
+        d    <NA>
+        e    <NA>
+        dtype: int64
+        >>> a.multiply(b, fill_value=0)
+        a       1
+        b       0
+        c       0
+        d       0
+        e    <NA>
+        dtype: int64""",
+        )
+    )
+    def multiply(self, other, axis, level=None, fill_value=None):  # noqa: D102
+        if level is not None:
+            raise NotImplementedError("level parameter is not supported yet.")
+
+        return self._binaryop(other, "__mul__", fill_value)
+
+    mul = multiply
+
+    @_cudf_nvtx_annotate
+    @docutils.doc_apply(
+        doc_binop_template.format(
+            operation="Multiplication",
+            op_name="rmul",
+            equivalent_op="other * frame",
+            df_op_example=""">>> df.rmul(1)
+                   angles  degrees
+        circle          0      360
+        triangle        3      180
+        rectangle       4      360
+            """,
+            ser_op_example=""">>> a.rmul(b)
+        a       1
+        b    <NA>
+        c    <NA>
+        d    <NA>
+        e    <NA>
+        dtype: int64
+        >>> a.multiply(b, fill_value=0)
+        a       1
+        b       0
+        c       0
+        d       0
+        e    <NA>
+        dtype: int64""",
+        )
+    )
+    def rmul(self, other, axis, level=None, fill_value=None):  # noqa: D102
+        if level is not None:
+            raise NotImplementedError("level parameter is not supported yet.")
+
+        return self._binaryop(other, "__rmul__", fill_value)
+
+    @_cudf_nvtx_annotate
+    @docutils.doc_apply(
+        doc_binop_template.format(
+            operation="Modulo",
+            op_name="mod",
+            equivalent_op="frame % other",
+            df_op_example=""">>> df.mod(1)
+                   angles  degrees
+        circle          0        0
+        triangle        0        0
+        rectangle       0        0
+            """,
+            ser_op_example=""">>> a.mod(b)
+        a       0
+        b    <NA>
+        c    <NA>
+        d    <NA>
+        e    <NA>
+        dtype: int64
+        >>> a.mod(b, fill_value=0)
+        a             0
+        b    4294967295
+        c    4294967295
+        d             0
+        e          <NA>
+        dtype: int64""",
+        )
+    )
+    def mod(self, other, axis, level=None, fill_value=None):  # noqa: D102
+        if level is not None:
+            raise NotImplementedError("level parameter is not supported yet.")
+
+        return self._binaryop(other, "__mod__", fill_value)
+
+    @_cudf_nvtx_annotate
+    @docutils.doc_apply(
+        doc_binop_template.format(
+            operation="Modulo",
+            op_name="rmod",
+            equivalent_op="other % frame",
+            df_op_example=""">>> df.rmod(1)
```

Comment:
is it possible to use `textwrap.dedent` and still get the right docstring formatting? If it's possible these will look more readable with proper indentations.

Reply:
Great call, yes that should be possible.